inst/rmarkdown/templates/frontiers_article/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,169 @@
+---
+title: "Article Title"
+runtitle: "Short Title"
+author:
+  - name: First Author
+    affiliation: '1'
+    etal: LASTNAME # First author's last name. 
+  - name: Co-Author
+    affiliation: '2'
+  - name: Co-Author
+    affiliation: '1,3'
+    email: email@uni.edu # Indicates corresponding Author
+    institution: Laboratory X, Institute X, Department X, Organization X
+    street: Street X
+    city: City X
+    state: State XX # only USA, Australia, Canada
+    zip: Zip Code X
+    country: Country X
+affiliation:
+  - id: '1'
+    department: Department X
+    institution: Institution X
+    city: City X
+    state: State XX # only USA, Australia, Canada
+    country: Country X
+  - id: '2' 
+    department: Department X
+    institution: Institution X
+    city: City X
+    state: State XX # only USA, Australia, Canada
+    country: Country X
+  - id: '3' 
+    department: Department X
+    institution: Institution X
+    city: City X
+    state: State XX # only USA, Australia, Canada
+    country: Country X
+output: rticles::frontiers_article
+csl: frontiers.csl
+bibliography: test.bib
```

Comment:
You actual filename is currently `test.BIB`.

Reply:
This is a git/OSX issue.  Renamed `test.bib` on machine, but doesn't reflect in the git push.  Will use little workaround

Reply:
See https://github.com/rstudio/rticles/pull/211/files#diff-7e36ef50a8912f76e9c03cfbb5fb698a